code/controllers/subsystem/shuttle.dm

```diff
@@ -461,6 +461,30 @@ SUBSYSTEM_DEF(shuttle)
 						. = TRUE
 						break
 
+		if("owner")
+			var/obj/docking_port/mobile/port = locate(params["id"]) in mobile
+			if(!port || !port.current_ship)
+				return
+			var/datum/overmap/ship/controlled/port_ship = port.current_ship
+			var/datum/action/ship_owner/admin/owner_action = new(port_ship)
+			owner_action.Grant(user)
```

Comment:
Won't this end up giving admins another action every time, even if they already had it

Reply:
Huh maybe I should actually read the entire PR before making review comments